sway-lib-std/src/vec.sw

```diff
@@ -20,8 +20,8 @@ impl<T> RawVec<T> {
     }
 
     /// Creates a `RawVec` (on the heap) with exactly the capacity for a
-    /// `[T; capacity]`. This is equivalent to calling `RawVec::new` when
-    /// `capacity` is `0`.
+    /// `T; capacity`. This is equivalent to calling `RawVec::new` when
```

Comment:
Hrmm, we either need to put the `[...]` back, escaped maybe, or word this better.  `T; capacity` by itself doesn't make sense.

Reply:
To elaborate, the syntax used is unique to vecs so I can see this accidentally getting removed. Should be `[T; capacity]`, where `T` is the type and `capacity` is the size it can contain.